src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -80,6 +80,7 @@ export interface IMutationTableProps {
     initialSortDirection?:SortDirection;
     paginationProps?:IPaginationControlsProps;
     showCountHeader?:boolean;
+    columnVisibility?: {[columnId: string]: boolean};
```

Comment:
currently unused, but will need this one for https://github.com/cBioPortal/cbioportal/issues/4026